<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QLabel" name="lbl_Message">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>80</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Mensaje enviado</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Adrress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>350</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>197.0.0.1  1234</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_Close">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CERRAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
